Formula/microws.rb

```diff
@@ -0,0 +1,41 @@
+class Microws < Formula
+  desc "Tiny WebSockets"
+  homepage "https://github.com/uNetworking/uWebSockets"
+  url "https://github.com/uNetworking/uWebSockets/archive/v0.14.5.tar.gz"
+  sha256 "4e4a4bcde543baae57ca8f327304874d1bb1fb11b2c4f71618d2b3279003f2d5"
+
+  depends_on "libuv"
+  depends_on "openssl"
+
+  def install
+    system "make"
```

Comment:
is the separate `make` step actually required?

Reply:
Yes. Otherwise `make install` complains about files being missing (i.e. `make install` only installs and does not build/compile).